src/WpBookingsUiModule.php

```diff
@@ -497,15 +500,21 @@ function () use ($c) {
             }
         );
 
-        add_submenu_page(
+        $aboutContext = [
```

Comment:
I would prefer if the things that relate to the page itself would be in a separate method.